seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/pom.xml

```diff
@@ -30,6 +30,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
Why add it? I think is not necessary to use dependencyManagement in there

Reply:
> Why add it? I think is not necessary to use dependencyManagement in there

This ensures that the driver version is consistent and without redefining the version separately.